sdclientapi/__init__.py

```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# Copyright © 2018–2022 The Freedom of the Press Foundation.
```

Comment:
these two lines should be removed until we have consensus here, see https://github.com/freedomofpress/securedrop/issues/6219#issuecomment-1039769952